iwyu_output.h

```diff
@@ -216,6 +216,13 @@ class OneIncludeOrForwardDeclareLine {
   const clang::NamedDecl* fwd_decl_;
 };
 
+struct NamespaceDeclUse {
```

Comment:
Could you add a little doc comment to this type? Just something around how we delay uses of namespace decls and finalize them in ResolvePendingAnalysis.